### PR TITLE
Consistent settings for heap memory

### PR DESCRIPTION
**Pull request by szarnyasg**
Please review & let me know what you think.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,6 +54,16 @@ The `config` directory contains the configuration for the scripts:
     Timeout is only applied to the solutions' run phase (see `-m` for `run.py`), so it is not applied to e.g. the build phase (see `-b` for `run.py`).
 * `reporting.json` -- configuration for the visualization
 
+### Setting the heap size
+
+By default, Java tools run with a heap size - both minimum (`Xms`) and maximum (`Xmx`) - of 6GB.
+This can be replaced with the following script (sing [`ag`](https://geoff.greer.fm/ag/)).
```
Comment (agarciadom) [inline README.md]: sing?

**Review by agarciadom — COMMENTED** [review]
This seems reasonable for the sake of comparability, except for that odd bit of text ("sing").

I'm wondering if it'd make sense to actually enforce this in CI from now on. Thoughts?

**Comment by szarnyasg** [comment]
`using` fixing...

> I'm wondering if it'd make sense to actually enforce this in CI from now on. Thoughts?

What should we enforce in CI from this PR?

**Comment by agarciadom** [comment]
I mean, ensuring that `Xmx` and `Xms` run with the same values everywhere.

**Comment by szarnyasg** [comment]
I think that's the benchmark conductor's responsibility and we cannot enforce this is in CI.

**Comment by agarciadom** [comment]
OK then - merging as is!

**Pull request merged**